CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 IMPROVEMENTS:
 * `data/policy_document`: Add support for `patch` capability for vault-1.9+. ([#1238](https://github.com/hashicorp/terraform-provider-vault/pull/1238))
 * `resource/database_secret_backend_connection`: Add support for InfluxDB connections ([#1121](https://github.com/hashicorp/terraform-provider-vault/pull/1121))
+* `resource/generic_secret`: Add support for deleting key metadata and all version data for a secret ([#1254](https://github.com/hashicorp/terraform-provider-vault/pull/1254))
```

Comment:
Is `deleting key metadata` the same as deleting `all version data`?

Reply:
I took this from the Vault docs [here](https://www.vaultproject.io/api-docs/secret/kv/kv-v2#delete-metadata-and-all-versions): `"This endpoint permanently deletes the key metadata and all version data for the specified key"`

I can remove the key metadata part if it reveals too many implementation details

Reply:
Simplified in b4126dd